makefile

```
@@ -0,0 +1,22 @@
+
+install:
+	poetry install
+
+build:
+	poetry build
+
+format:pro_commit
+	poetry run black switchbot_api/
+	poetry run mypy switchbot_api/
+
+clean:
+	rm -rf dist
+
+test:test_update
+	poetry run pytest --cov vehicle tests
+
+test_update:
+	pytest --snapshot-update
+
+pro_commit:
```

Did you mean pre commit?
```suggestion
pre_commit:
```